tests/php/unit/AdvancedCustomFieldTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+# -*- coding: utf-8 -*-
+
+declare(strict_types=1);
+
+namespace MultisiteGlobalMedia\Tests\Unit;
+
+use Brain\Monkey\Functions;
+use MultisiteGlobalMedia\ACF\Image;
+use MultisiteGlobalMedia\Plugin;
+use MultisiteGlobalMedia\Site;
+use MultisiteGlobalMedia\SiteSwitcher;
+use MultisiteGlobalMedia\Tests\TestCase;
+
+define('ABSPATH', 1); // ACF quits early without it
+include_once dirname(__DIR__, 3) . '/vendor/wpackagist-plugin/advanced-custom-fields/includes/class-acf-data.php';
```

Comment:
@felly I do think that this kind of things must be under `self::setUp` but despite that, an unit test shouldn't introduce external code, it should mock it. So my question is, is the inclusion of such a file necessary for some reason that justify not mocking the functions?

Reply:
@widoz I am using the external code to create a mock of this imported class `\ACF_Data::class`. Is there a better way to do this?

Reply:
@felly For unit tests you can create a stub for that class that can be loaded during the bootstrap. For instance having a file `tests/stubs/acf.php`  within it you can define the class 

```
class ACF_Data {}
```

This should allow you to mock the class. You can even add the empty methods in it in case you have problems. 
But you can also do `$store = $this->createMock('\\ACF_Data');` without the need for a stub afaik.

Reply:
@widoz Thanks for the advice! I have created a stub because otherwise this warning was given:

```
Cannot stub or mock class or interface "\ACF_Data" which does not exist
```

Reply:
@felly Thanks so much for this.